lib/Graph/Graph.cpp

```diff
@@ -1021,6 +1021,14 @@ PowNode *Function::createPow(llvm::StringRef name, NodeValue Base, float exp) {
   return addNode(new PowNode(name, Base.getType(), Base, exp));
 }
 
+PowNode *Function::createSqrt(llvm::StringRef name, NodeValue Base) {
+  return addNode(new PowNode(name, Base.getType(), Base, 0.5f));
```

Comment:
nit: here and in `createReciprocal()`, I would just directly do `return createPow(name, base, 0.5f)`;

Reply:
I simplified further. Since there is no `reciprocal` and `sqrt` nodes in the IR, I now simply call `createPow` from the loader.